docker-compose.yml

```diff
@@ -24,3 +26,34 @@ services:
      - POSTGRES_PASSWORD=mysecretpassword
     ports:
      - "5432:5432"
+  mongodb:
+    image: mongo
```

Comment:
NB. I haven't locked the version for now but I could do if PR reviewers think it's valuable. My take is in a dev environment it's useful to have the latest version unless there's a specific need for an earlier version. However I also fully understand the view of having it mach the versions in prod. Happy to change.